### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'edit') 

### ISSUE THREAD

**Issue by Tetracyl** [Tetracyl/EarTensifier#217]
```14|EarTensifier  | [9/6/2022] [5:43:45 AM] [Shard 082] › ✖  error     TypeError: Cannot read properties of undefined (reading 'edit') 
14|EarTensifier  |     at Loop.run (/home/seif/ear_tensifier/src/components/buttons/Loop.js:48:40)
14|EarTensifier  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
14|EarTensifier  |     at async InteractionCreate.run (/home/seif/ear_tensifier/src/listeners/client/InteractionCreate.js:26:17)```

**Comment by Tetracyl** [comment]
```14|EarTensifier  | [9/6/2022] [5:44:00 AM] [Shard 022] › ✖  error     TypeError: Cannot read properties of null (reading 'edit') 
14|EarTensifier  |     at Pause.run (/home/seif/ear_tensifier/src/components/buttons/Pause.js:26:40)
14|EarTensifier  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
14|EarTensifier  |     at async InteractionCreate.run (/home/seif/ear_tensifier/src/listeners/client/InteractionCreate.js:26:17)```

**Comment by Tetracyl** [comment]
Fixed in 2.4.0
